micro-rdk-server/sdkconfig.defaults

```diff
@@ -41,3 +41,5 @@ CONFIG_ESP_COREDUMP_ENABLE=y
 CONFIG_ESP_COREDUMP_MAX_TASKS_NUM=64
 CONFIG_ESP_COREDUMP_DECODE="disable"
 
+# NTP Config
+CONFIG_LWIP_SNTP_UPDATE_DELAY=86400000
```

Comment:
1 day in ms

Reply:
That feels much too long. The default is an hour. Why change it/

Reply:
there shouldn't be much drift at run time. It's mostly during deep sleep that the clock we incur a lot of errors.

Reply:
But I wonder if when the wifi we are connected to doesn't have internet if failing to sync time will schedule the next sync to +24h

Reply:
then having the default 1hr would be better, no? Then the gap would be smaller in this case.

Another thing I'm not sure about is, if the device is waking only for brief periods of time, will the time sync get called during that small timeframe.